meson.build

```diff
@@ -128,6 +128,15 @@ grcli_exe = executable(
 
 install_headers(api_headers)
 
+# generate grout.pc for those headers
+import('pkgconfig').generate(
+  name: 'grout',
+  description: 'Helpers for grout integration and management',
+  version: meson.project_version(),
```

Comment:
This is redundant. It is already the default value when version is not specified.